### PR TITLE
Added gulpfile.js

### PR DESCRIPTION
**Pull request by ayushinigam**
gulpfile.js has been added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-[![Stories in Ready](https://badge.waffle.io/MDLTechUniversity/panpage.png?label=ready&title=Ready)](https://waffle.io/MDLTechUniversity/panpage)
```
Comment (skeep) [inline README.md]: why u r deleting this file content?

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+//build task
+gulp.task('copy:dist', function (){
+  return gulp.src(['.tmp/**/*.css','.tmp/**/*.min.js','.tmp/**/*.html'])
+    .pipe(gulp.dest('.dist'));
+});
+
+
+
+//clean
+gulp.task('clean', function (cb) {
```
Comment (skeep) [inline gulpfile.js]: whats cb? callback function?

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  del([
+    'www/**/*'
+  ], cb);
+});
+
+
+ gulp.task('serve', gulpSequence(['scripts', 'lint','css','minify','jade']));
+// gulp.task('dev', gulpSequence(['browser-sync','watch']));
+ gulp.task('build', gulpSequence(['clean:dist','copy:dist']));
+ gulp.task('gulp-sequence', gulpSequence('serve', 'build'));
```
Comment (skeep) [inline gulpfile.js]: will you run gulp-sequesce as a task?

File: gulpfile.js (hunk 2 above)
Comment (ayushinigam) [inline gulpfile.js]: Yes it is the call back function

File: gulpfile.js (hunk 3 above)
Comment (ayushinigam) [inline gulpfile.js]: I am using gulp-sequence to run certain task in serial

**Pull request closed without merging**